src/TcUnit.TestAdapter/Models/PlcProject.cs

```diff
@@ -73,36 +103,41 @@ private void ParsePOUs ()
                 }
             }
 
-            var plcReferences = doc.Elements(XmlNamespace + "Project")
-                                            .Elements(XmlNamespace + "ItemGroup")
-                                            .Elements(XmlNamespace + "PlaceholderReference");
+            // the following supports two different types of library references which may be used in a PLC project
+            XmlNamespaceManager ns = new XmlNamespaceManager(new NameTable());
+            ns.AddNamespace("prefix", XmlNamespace.NamespaceName);
+            var xpath = "//prefix:Project/prefix:ItemGroup/prefix:LibraryReference";
+            xpath += "|//prefix:Project/prefix:ItemGroup/prefix:PlaceholderReference";
+
+            var plcReferences = doc.XPathSelectElements(xpath, ns);
 
             foreach (var plcLibrary in plcReferences)
             {
                 var name = plcLibrary.Attribute("Include")?.Value;
 
                 var reference = new PlcLibraryReference
                 {
-                    Name = name,
+                    Name = name.Split(',')[0],
                 };
 
-                var parameters = plcLibrary.Element(XmlNamespace + "Parameters");
+                var parameters = plcLibrary.ElementAnyNS("Parameters");
```

Comment:
Sometimes my projects contained an `xmlns=''` in the definition of parameters, for no apparent reason. This extension method ignores the namespace.

```
      <Parameters>
        <Parameter ListName="GVL_PARAM_TCUNIT" xmlns="">
          <Key>MAXNUMBEROFASSERTSFOREACHTESTSUITE</Key>
          <Value>2000</Value>
        </Parameter>
```